source/includes/_resource.collections.md

```diff
@@ -715,6 +717,55 @@ $ curl -X PUT "https://api.vhx.tv/collections/1" \
   </tbody>
 </table>
 
+<h3 class="text-2 head-4 text--navy text--bold is-api margin-top-large margin-bottom-medium" id="collection-position-update">Update a Collection's Position</h3>
+
+> <h5 class="head-5 text--white margin-bottom-medium">Update a Collection's Position</h5>
+
+> Definition
+
+```shell
+PUT /collections/:id/update_position
+```
+
+> Example Request
+
+```shell
+$ curl -X PUT "https://api.vhx.tv/collections/1/update_position" \
+  -H "Content-Type: application/json" \
+  -d '{"position": 3}' \
+  -u o3g_4jLU-rxHpc9rsoh3DHfpsq1L6oyM:
```

Comment:
I see that we already have a pattern of using an actual token sort of value for the documentation, but shouldn't we use some kind of description for this? Like `${USER_ACCESS_TOKEN}` or something of that sort? I feel like if I am a reader with no context, I don't know what that value represents.

Reply:
The API documentation does leave a lot to be desired, but this is sort of covered in the Authentication section: https://dev.vhx.tv/docs/api/#authentication. I also think it's fair to assume that the reader knows that the curl `-u` flag is the user information for Basic Auth in this format `username:password`.